src/frob.cc

```diff
@@ -28,7 +28,7 @@
 #include <Python.h>
 #include <frameobject.h>
 
-#if PYFLAME_PY_VERSION == 3
+#if PYFLAME_PY_VERSION == 3 || PYFLAME_PY_VERSION == 36
```

Comment:
let's set change the existing references from `PYFLAME_PY_VERSION = 2` to `PYFLAME_PY_VERSION = 20`, and `PYFLAME_PY_VERSION = 3` to `PYFLAME_PY_VERSION = 30`. That way this `#if` can become:

```c
#if PYFLAME_PY_VERSION >= 30
```